src/core/scene/a-scene.js

```diff
@@ -343,9 +302,6 @@ module.exports.AScene = registerElement('a-scene', {
           if (!self.isMobile && !self.checkHeadsetConnected()) {
             requestFullscreen(self.canvas);
           }
-
-          self.renderer.setAnimationLoop(self.render);
-          self.resize();
         }
 
         function enterVRFailure (err) {
```

Comment:
We should pull out all the VR handling and drastically simplify. There are lots of codepaths for scenarios that we almost certainly do not care about supporting anymore. I tested functionality on Quest but did not want to dig into further cleaning this up in this PR.